.hooks/README.md

```diff
@@ -0,0 +1,17 @@
+# Git Hooks
+
+Run conventional commit checks locally without any tool installation
+
+## Usage
+
+Execute the below command in the root directory of the project:
+
+```sh
+ln -s -f ../../.hooks/commit-msg ./.git/hooks/commit-msg
```

Comment:
Where should `../..` point to? Since this moves out of the repository it seems a bit odd (and please don't put `--force` in there).

What about only configuring the local git in this repo? This can be achieved by running `git config core.hooksPath .hooks` in the root of the repo and should work like a charm. Alternatively git uses `.git/hooks` by default so you could symlink the hooks to there.

```suggestion
ln -s .hooks/commit-msg .git/hooks/commit-msg
```

```suggestion
git config core.hooksPath .hooks
```

Reply:
I also felt it odd from path perspective. It does not work (gives error) if I don't give the path like that. Did you try it yourself? 
`-f` option is used because `commit-msg` file does not exist in .git/hooks, otherwise one needs to `touch` the file and then link without `-f`. 
We could also use hooks path but then we don't know how individuals' `.git` is configured. In either cases we will still need to symlink. 
So considering these, I preferred to keep it simple where one needs to just run the `ln` command.

Reply:
I'm assuming that you have configured the hooks for your system centrally which is why the local ones don't work. What does the output of `git config core.hooksPath` show inside and outside the repository?

Force should not be needed if the file does not exist, according to the [man page](https://man.archlinux.org/man/ln.1#f) it is only required if you wish to overwrite the destination file.

I think the git-config approach (my second suggestion) should be the most reliable, it overwrites the global configuration of the user but that's just how these hooks work. You only have to make sure to run the command inside the repository so that it is only configured there and not somewhere else.